### PR TITLE
Akhil forgotpassword

### PR DESCRIPTION
**Pull request by Aillgates**
. Create a "Forgot Password" page with form validation.

. Build a reusable modal for success/error messages.

. password reset page

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import { Routes, Route } from 'react-router-dom'
-import UserRoutes from './routes/UserRoutes';
+import { Routes, Route } from "react-router-dom";
+import ProviderRoutes from "./routes/ProviderRoutes.jsx";
+import Hello from "./Hello.jsx";
```
Comment (JaseelaAlimahamed) [inline src/App.jsx]: ./hello.jsx means?

**Review by JaseelaAlimahamed — APPROVED** [review]

File: src/routes/ProviderRoutes.jsx [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import React from 'react'
 import { Route, Routes, Navigate } from 'react-router-dom'
+import NewPassword from '../pages/NewPassword'
+import ForgotPassword from '../pages/ForgotPassword'
+import OTPPage from '../pages/ForgotOtp'
```
Comment (JaseelaAlimahamed) [inline src/routes/ProviderRoutes.jsx:5]: why forgot otp
?????

Comment (Aillgates) [reply src/routes/ProviderRoutes.jsx:5]: I have developed a 'Forgot OTP Password' page, and now realize it should be rendered upon clicking the designated button.

Comment (JaseelaAlimahamed) [reply src/routes/ProviderRoutes.jsx:5]: forgot otp fuction is not there either forgot password or resend otp

File: src/components/reUsableComponents/ButtonComponent.jsx [hunk 3]
```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 
-const ButtonComponent = ({ label, type = 'button', disabled = false, btnWidth = 'w-auto', btnHeight = 'h-auto' }) => {
+
+const ButtonComponent = ({ label, type, disabled, btnWidth , btnHeight , handleSubmit }) => {
```
Comment (JaseelaAlimahamed) [inline src/components/reUsableComponents/ButtonComponent.jsx:4]: btnWidth  btnHeigh why you change this reusable feeatures in button component

**Review by JaseelaAlimahamed — APPROVED** [review]

**Pull request merged**